### PR TITLE
Add chat page

### LINKED ISSUE #865: Community Support + Gitter/Discord
I like this project and I would like to contribute.

However, I noticed that there is no community channel such as Gitter or slack so the developers can discuss PRs and other issues. Discussing ideas through GitHub is cumbersome.

I propose to setup a community chat for this as it will help with collaborating and assisting users.

https://gitter.im/
https://discordapp.com/

### PR DESCRIPTION
**Pull request by rpkak**
Added discord for RQ and a page of the website to join.

closes #865

### CODE REVIEW

File: docs/chat/index.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+---
+title: "RQ Discord"
+layout: chat
+---
+
+Join our discord [here](/chat/join) if you need help or want to chat about contributions or what should come next in RQ.
+
+<iframe src="https://discord.com/widget?id=844816706231861248&theme=dark" height="500" allowtransparency="true" frameborder="0" sandbox="allow-popups allow-popups-to-escape-sandbox allow-same-origin allow-scripts" style="width: 100%;"></iframe>
```
Comment (selwin) [inline docs/chat/index.md:8]: Sorry for being nitpicky, can we eliminate the use of iframe and JS here? A simple `href` that opens in a new tab will do.

Comment (rpkak) [reply docs/chat/index.md:8]: This iframe is a widget I copied from discord. It is not intended for a new tab. It is not so important so I can delete it. Shall I?

Comment (selwin) [reply docs/chat/index.md:8]: Yes please, you can also just turn there `[here]` text into a hyperlink so we can get rid of `docs/chat/join.html`.

Comment (rpkak) [reply docs/chat/index.md:8]: Ok, I made `chat/join` so there is a readable link for the discord, but I remove it.

**Comment by rpkak** [comment]
Any changes for this PR?

**Comment by selwin** [comment]
Sorry I haven’t had much time to review PRs. Will try to take a look this
weekend.

Win~

**Comment by rpkak** [comment]
@selwin do you want to join the discord to add some webhooks so releases etc. will be automatically displayed.

**Comment by rpkak** [comment]
ping

**Comment by selwin** [comment]
@rpkak just joined the Discord. I've only used Discord once so I have no idea what to do lol.

**Comment by rpkak** [comment]
@selwin So to [this](https://github.com/rq/rq/pull/1474#issuecomment-873006255):

Go in discord to server settimgs -> integrations -> Create Webhook.

Then set the webhook name to GitHub and the channel to announcements and save the workflow.

copy the webhook url and to to GitHub:

settings of the repo -> Webhooks -> Add Webhook.

Payload URL is the webhook url you from discord but `/github` in the end.
Content Type is `application/json`.
Secret should be empty.

Then use `Let me select individual events.`, deselect push and select release.

**Comment by selwin** [comment]
@rpkak I experimentally enabled [RQ's Github discussions](https://github.com/rq/rq/discussions). Have you used it before? I'm wondering if this can be used for community support questions.

If this works out well, I can give a few people moderator privileges to help out.

**Comment by rpkak** [comment]
I didn't used this before.

**Pull request merged**